sdk/identity/azure-identity/src/client_certificate_credential.cpp

```diff
@@ -448,9 +481,20 @@ ClientCertificateCredential::ClientCertificateCredential(
     thumbprintBase64Str = Base64Url::Base64UrlEncode(ToUInt8Vector(mdVec));
   }
 
+  std::string x5cHeaderParam{};
+  if (sendCertificateChain)
+  {
+    // Since there is only one base64 encoded cert string, it can be written as a JSON string rather
```

Comment:
The x5c itself is clearly defined as an array in the specs. So, your implementation shall still generate a list even if it contains only one node, e.g. `x5c = ["a leaf cert"]`.

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-cpp/pull/6052